posts/2022-01-18-ga-release-post.adoc

```diff
@@ -0,0 +1,413 @@
+
+[source]
+----
+gpg2 --verify openliberty-runtime-22.0.0.1.zip.asc
+If the verification is successful, the command produces console output that is similar to the following example.
+
+gpg: assuming signed data in 'openliberty-runtime-22.0.0.1.zip'
+gpg: Signature made Wed Nov 24 09:02:44 2021 EST
+gpg: using RSA key 91FFD9A642D060B66B802B9D4D210F6946102B8E
+gpg: Good signature from "WebSphere Liberty (Works for IBM) contbld@uk.ibm.com" [ultimate]
+gpg: aka "WebSphereLiberty" [ultimate]
+----
+
+[#expansion_variables]
+== Using expansion variables in server.env
+
+This update allows you to specify environment variables in the server.env file on Linux which are resolved when the server starts. This capability already exists on Windows.
+
+Prior to this update, it might be necessary to provide custom packaged servers with hard-coded values in the server.env. Now server.env can consume these values from the system environment.
+On Windows this capability already existed, though it was not documented anywhere.
+
+On Windows, this capability is enabled by default and has always existed. Since this is new to all other operating systems, it is necessary to enable the new capability by adding a comment line near the top of the file:
+
+`# enable_variable_expansion`
+
+Environment variables are specified using `${variable_name}` syntax, except on Windows where the syntax is `!variable_name!`
+In the examples below the `LOG_FILE` variable is assigned the value of an environment variable. This example changes the name of the log file from the default name of `console.log`.
+
+Example `server.env` on Windows:
+
+[source]
+----
+keystore_password=XASEvZMKn3wG6XuTaVYFr8C
+LOG_FILE=!CONSOLE_LOG!
+----
+
+Example `server.env` on Linux:
+
+[source]
+----
+# enable_variable_expansion
+keystore_password=XASEvZMKn3wG6XuTaVYFr8C
+LOG_FILE=${CONSOLE_LOG}
+----
+
+For more information check out the link:https://openliberty.io/docs/21.0.0.11/reference/config/server-configuration-overview.html[Server configuration overview] and link: https://www.ibm.com/docs/en/was-liberty/core?topic=manually-customizing-liberty-environment[Customizing the Liberty environment] documentation.
+
+[#jvm]
+== JVM Working Directory
+
+This enhancement would allow the user to specify the `SERVER_WORKING_DIR` environment variable and set the `JVM` working directory location to something other than the `${WLP_OUTPUT_DIR}/serverName location`. The request from the `RFE` is to NOT make this a fully qualified path for portability purposes. Thus a path that is relative to the `${WLP_OUTPUT_DIR}/serverName` location can be used, as well as, an absolute path(one that contains a `c:\` on Windows, or a `/` on linux based operating systems).
+
+So for example, the user could set the Open Liberty `JVM` output to be added to the `${WLP_OUTPUT_DIR}/serverName/logs` location so that all `JVM` related data would be in with the server log data (relative path example):
+
+`SERVER_WORKING_DIR=logs`
```

Comment:
```suggestion
For example, the user could set the Open Liberty `JVM` output to be added to the `${WLP_OUTPUT_DIR}/serverName/logs` location so that all `JVM` related data would be in with the server log data (relative path example) by setting the following:

`SERVER_WORKING_DIR=logs`
```